### PR TITLE
Update build to remove manual steps

### PR DESCRIPTION
**Pull request by tejohnso**
- build using build.sh
 - e2e tests
 - copy build output (sh,exe) to master/
 - sign exes
 - fetch json config files and update version and rollout%
 - copy updated json files to master/
 - set acl and metadata in master/
 - backup production
 - copy master/ to production
 - copy master/ to releases/[version]

@fjvallarino @ahmedalsudani please review

### CODE REVIEW

**Comment by fjvallarino** [comment]
:+1:

File: circle.yml [hunk 1]
```diff
@@ -57,46 +51,24 @@ deployment:
...
-      - gsutil setmeta -h "Cache-Control:private, max-age=0" gs://install-versions.risevision.com/master/*.exe
-      - gsutil setmeta -h "Cache-Control:private, max-age=0" gs://install-versions.risevision.com/master/electron*.json
-      - gsutil acl ch -u AllUsers:R gs://install-versions.risevision.com/master/*.sh
-      - gsutil acl ch -u AllUsers:R gs://install-versions.risevision.com/master/*.exe
-      - gsutil acl ch -u AllUsers:R gs://install-versions.risevision.com/master/electron*.json
+      - gsutil cp builds/*.{sh,exe} gs://install-versions.risevision.com/master/
+      - curl -vv 216.21.167.45:7790/sign
+      - gsutil cp gs://install-versions.risevision.com/electron*.json .
+      - for f in electron*.json; do sed -i "s/\"InstallerElectronVersion.*/\"InstallerElectronVersion\":$(cat version.json),/" $f;done
+      - for f in electron*.json; do sed -i 's/"LatestRolloutPercent.*/"LatestRolloutPercent":2,/' $f;done
```
Comment (ahmedalsudani) [inline circle.yml]: @tejohnso I know we agreed on 2% rollout percentage, but lately we've been starting as low as 1. Should this stay as 2?

File: circle.yml [hunk 2]
```diff
@@ -57,46 +51,24 @@ deployment:
...
+      - gsutil cp electron*.json gs://install-versions.risevision.com/master/
+      - gsutil acl ch -u AllUsers:R gs://install-versions.risevision.com/master/*.{sh,exe,json}
+      - gsutil setmeta -h "Cache-Control:private, max-age=0" gs://install-versions.risevision.com/master/*.{sh,exe,json}
+      - gsutil setmeta -h "Content-Disposition:attachment" gs://install-versions.risevision.com/master/*.sh
+      - gsutil cp -p gs://install-versions.risevision.com/*.{exe,sh,json} gs://install-versions.risevision.com/backups/$(cat version.txt)/
+      - gsutil cp -p -m gs://install-versions.risevision.com/master/* gs://install-versions.risevision.com/
```
Comment (ahmedalsudani) [inline circle.yml]: @tejohnso I think `gsutil cp` does not copy the ACL metadata, so we need to either change the bucket default or set ACL after copying

File: circle.yml (hunk 1 above)
Comment (tejohnso) [inline circle.yml]: @ahmedalsudani 2 is fine since we'll be making small changes.  Easy to modify if need be later on.

File: circle.yml (hunk 2 above)
Comment (tejohnso) [inline circle.yml]: @ahmedalsudani metadata is retained in cloud to cloud copies.

**Comment by ahmedalsudani** [comment]
:+1:

**Pull request merged**